### DISCUSSION TITLE
rna_denovo: Combine multiple silent files and cluster

### DISCUSSION THREAD

**Discussion by mandar5335** [RosettaCommons/rosetta#121] [Rosetta] [answered]
Hi,

I am running _rna_denovo_ and planned to generate multiple silent files for same RNA molecule. 

How can I combine these multiple silent files, with unique tags to each frame? 

Best,
Mandar

**Comment by roccomoretti** [comment, marked as answer]
Silent files should be set up such that you can just concatenate the file and have them work. Rosetta will automatically unique-ify the tags when it reads in the file.

If that doesn't work for your pipeline (e.g. if you want to know which tags Rosetta will use for which structure to extract just the top-scoring ones), there's a `combine_silent` application. Just pass the multiple silent files with `-in:file:silent` and the name of the new silent file with `-out:file:silent` and Rosetta will read in all the silent files, unique-ify the tags and then write them out again.

**Reply by mandar5335** [reply]
thanks!
